examples/nlp/machine_translation/megatron_nmt_training.py

```diff
@@ -163,6 +167,10 @@ def main(cfg) -> None:
             # Optimizer overrides.
             pretrained_cfg.optim = cfg.model.optim
 
+            # override hidden_size
+            if hasattr(pretrained_cfg.encoder, 'hidden_size'):
+                pretrained_cfg.hidden_size = pretrained_cfg.encoder.hidden_size
```

Comment:
Don't we need this for the decoder as well?

Reply:
Yes, I wasn't sure how to handle this since the MegatronNMT class only looks for 1 hidden_size value and I think it may not support encoder/decoder with different hidden sizes.